internal/clients/team4/utilities.go

```diff
@@ -100,27 +145,32 @@ func (c *client) getJudge() shared.ClientID {
 	return 0
 }
 
-// func dump(filename string, format string, v ...interface{}) {
-// 	//f, err := os.Create(filename)
-// 	f, err := os.OpenFile(filename, os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
-
-// 	if err != nil {
-// 		log.Fatal(err)
-// 	}
-
-// 	defer f.Close()
-
-// 	_, err2 := f.WriteString(fmt.Sprintf(format, v...))
-
-// 	if err2 != nil {
-// 		log.Fatal(err2)
-// 	}
-
-// }
 func boolToFloat(input bool) float64 {
 	if input {
 		return 1
 	}
 	return 0
 }
 
+func checkIfClientIsInList(lst []shared.ClientID, c shared.ClientID) bool {
+	for _, e := range lst {
+		if e == c {
+			return true
+		}
+	}
+	return false
+}
+
+func createClientSet(lst []shared.ClientID) []shared.ClientID {
+	uniqueMap := make(map[shared.ClientID]bool)
+	var uniqueLst []shared.ClientID
+	for _, e := range lst {
+
+		_, ok := uniqueMap[e]
+		if !ok {
+			uniqueMap[e] = true
+			uniqueLst = append(uniqueLst, e)
+		}
```

Comment:
I think this would also work and looks better
```golang
if _, ok := uniqueMap[e]; !ok {
	uniqueMap[e] = true
	uniqueLst = append(uniqueLst, e)
}
```

Reply:
@Spruudis